lib/players/computer.rb

```diff
@@ -0,0 +1,85 @@
+# require_relative "../board.rb"
+# require_relative "../player.rb"
+# require_relative "../players/human.rb"
+# require_relative "../game.rb"
+# require 'pry'
+
+
+class Computer < Player
+	WIN_COMBINATIONS = [
+	  [0,1,2],
+	  [0,1,2],
+      [3,4,5],
+      [6,7,8],
+      [0,3,6],
+      [1,4,7],
+      [2,5,8],
+      [0,4,8],
+      [6,4,2] ]
+
+	def move(board)
+		if winning_move(board)
+			winning_move(board)
+		elsif blocking_move(board)
+			blocking_move(board)
+		elsif board.cells[4] == " "
+			"5"
+		elsif board.cells[0] == " "
+			"1"
+		elsif board.cells[2] == " "
+			"3"
+		elsif board.cells[6] == " "
+			"7"
+		elsif board.cells[8] == " "
+			"9"
+		else	
+			rand(1..9).to_s
+		end		
```

Comment:
You could avoid the duplicate method calls with something like
```
winning_move(board) || blocking_move(board) || other_move(board)
```
and moving the rest of the `elsif` logic into the new `other_move` method. If `winning_move` or `blocking_move` return nil, the next method in the chain will be evaluated; but if they return a truthy value, `move` will exit and return that value.